source/extensions/filters/network/well_known_names.h

```diff
@@ -59,6 +59,9 @@ class NetworkFilterNameValues {
   const std::string ZooKeeperProxy = "envoy.filters.network.zookeeper_proxy";
   // WebAssembly filter
   const std::string Wasm = "envoy.filters.network.wasm";
+  // A common namespace which will propagated the dynamic metadata from
+  // downstream to upstream
+  const std::string DownToUp = "envoy.common.down_up";
```

Comment:
Naming suggestion: `shared_with_upstream`? Would be good to align with the naming choice for the filter state (https://github.com/envoyproxy/envoy/pull/21700/files).

Reply:
will change